frontend/app/services/TouchpointBackend.scala

```diff
@@ -60,10 +65,9 @@ object TouchpointBackend {
 
 case class TouchpointBackend(memberRepository: FrontendMemberRepository,
                              stripeService: StripeService,
-                             zuoraSoapService: ZuoraSoapService,
-                             zuoraRestService: ZuoraRestService) extends ActivityTracking {
-
-  val subscriptionService = new SubscriptionService(zuoraSoapService, zuoraRestService)
+			     zuoraSoapClient: soap.ClientWithFeatureSupplier,
+			     zuoraRestClient: rest.Client,
```

Comment:
This looks to be a new coding style, and I don't quite see the benefits of it. It is technically possible that the client code can miss this package prefix and can make the code a lot less readable, especially since there are multiple `Client` classes in our codebase.